sonarqube-companion-rest/src/main/java/pl/consdata/ico/sqcompanion/sync/SyncAfterAppInitialization.java

```diff
@@ -20,6 +21,9 @@ public SyncAfterAppInitialization(final SynchronizationService synchronizationSe
 		this.synchronizationService = synchronizationService;
 	}
 
+	/**
+	 * TODO (mp): to nie powinno blokować wstawania aplikacji
```

Comment:
przy aktualnej domyślnej konfiguracji baza trzyma stan pomiędzy uruchomieniami aplikacji (więc jak masz plik na zewnętrznym zasobie to dane już są). W kodzie się postarałem o to żeby ta operacja nie blokowała wstawania a na froncie masz progressbar pokazujący ile danych już się zsynchronizowało - IMO od strony użytkownika takie zachowanie jest przyjaźniejsze